### DISCUSSION TITLE
Probleme beim Drescher abtanken

### DISCUSSION THREAD

**Discussion by Risca81** [Courseplay/courseplay#6844] [Q&A] [answered]
Guten Morgen!

Zuerst einmal vielen Dank für dieses tolle Script für den LS!

Ich habe seit kurzem festgestellt, dass beim Abtanken des Dreschers der Abfahrer des Öfteren in den Drescher fährt, also stumpf nach rechts abbiegt, sobald der Abtankvorgang abgeschlossen ist. Ganz oft passiert es, wenn der Drescher beim Abtanken weiter fährt..aber auch, wenn er anhält.

Das Problem habe ich bei Abtanken mit Anhänger und Überladewagen. 

Ich habe Fruchtumfahrung und Wenden im Feld deaktiviert. 

Meine Version ist:  6.03.00040

Was benötigt ihr noch für Informationen?

Grüße

**Comment by Tensuko** [comment, marked as answer]
Fruchtumfahrung auf aktiviert schalten! Wenden auf dem Feld hat eh keinen Einfluss auf den Mode2, das wäre egal.

Noch mal hier damit es alle lesen können, aber bitte nicht Discussion UND Issue für die gleiche Anfrage aufmachen, danke.

**Reply by Risca81** [reply]
Hallo.
Funktioniert soweit, danke!

Aber: nach einer gewissen Zeit fährt der Abfahrer nicht mehr ab. Wenn ich ihm eine Haltezeit gebe, was vorher ging, dann behält er diese nicht mehr. Habe in einem YouTube Video gesehen, dass diese nötig sei. Funktioniert auch die ersten Male ganz gut, aber dann halt nimmer.

LG

**Reply by Tensuko** [reply]
Eine Haltezeit im Mode2 ist nicht vorgesehen und wird auch nicht benötigt.
